site/_posts/2020-12-08-xkit-and-fusionauth.md

```diff
@@ -0,0 +1,107 @@
+
+Once this is installed you have a full featured user management system, ready to go. APIs to control everything, multi factor authentication, consent management, SAML, OIDC, and more, hosted wherever you want. There's also FusionAuth cloud, a managed services offering, if you don't want to host FusionAuth yourself.
+
+### Install Xkit
+
+After you're set up with FusionAuth you'll want to head over and [create your Xkit account](https://app.xkit.co/sign-up). Upon sign-up, Xkit will also prompt you with some basic information needed to set up the environment. Fill out those details and you're good to go there.
+
+To set up Xkit in your code base, you'll need to add the script tag for `xkit.js` on your front-end:
+
+```html
+<script src="https://<your-slug>.xkit.co/xkit.js"></script>
+```
+
+In addition, your users need a place to actually sign in to their apps, such as Dropbox for your cooking recipe sharers. The easiest way to do this is to direct users to the hosted integration catalog Xkit has set up for you:
+
+```html
+<a href="https://<your-slug>.xkit.co">Integration Catalog</a>
+```
+Alternatively, if you need more customization, you can embed Xkit's catalog on your site and customize its styling to fit your look and feel ([details here](https://docs.xkit.co/docs/self-hosted-catalog)). If you still need more flexibility, you can use the SDK.
+
+### Connect FusionAuth with Xkit
+
+Now that you have both FusionAuth and Xkit set up, you'll need to connect the two. We do this by collecting some information from your FusionAuth dashboard and inputting it into Xkit. 
+
+Specifically, you'll need to generate an RSA key and then add your "iss" claim, "aud" claim and JWKS URL into your Xkit account. This setup is [fully documented](https://docs.xkit.co/docs/fusionauth).
+
+{% include _image.liquid src="/assets/img/blogs/xkit-fusionauth-integration/rsa-keypair.png" alt="The generated RSA key for use with Xkit." class="img-fluid" figure=false %}
+
+Once you've done that and clicked save, FusionAuth and Xk9it will be connected. Huzzah!
+
+You can now have your user log in using FusionAuth's Login API, federated with Active Directory, Google or Facebook, or an OAuth authorization code flow. You'll get a JWT from FusionAuth on successful user authentication. This JWT can then be sent to Xkit to authenticate the user in Xkit, and therefore grant them access to integrations you've configured.
```

Comment:
Should "You can now have your user log in using FusionAuth's Login API" be "You can now have your users log in using FusionAuth's Login API" (with the s on user)?